.travis.yml

```diff
@@ -1,22 +1,62 @@
 language: scala
-scala:
-  - 2.12.10
-  - 2.13.1
 
 git:
   depth: false # Avoid sbt-dynver not seeing the tag
 
 env:
-  - TRAVIS_JDK=8
   - TRAVIS_JDK=11
 
 before_install: curl -Ls https://raw.githubusercontent.com/shyiko/jabba/0.11.2/install.sh | bash && . ~/.jabba/jabba.sh
 install: jabba install "adopt@~1.$TRAVIS_JDK.0-0" && jabba use "$_" && java -Xmx32m -version
 
-script:
-  - ./scripts/test-code.sh
-  - ./scripts/validate-code.sh
-  - ./scripts/validate-docs.sh
+jobs:
+  include:
+    - stage: validations
+      script: scripts/validate-code.sh
+      name: "Code validations (headerCheck, scalafmt, mima)"
+   
+    - stage: test-jdk11
+      name: "Run tests with Scala 2.10 and AdoptOpenJDK 11"
+      script: scripts/test-code.sh
+      env: 
+        - TRAVIS_SCALA_VERSION=2.10.7 
+        - TRAVIS_JDK=11 
```

Comment:
Wow, does Twirl 1.5 still do Scala 2.10? 
Is Scala 2.10 even expected to work on JDK11?

Reply:
Only the plugin because of sbt 0.13, but indeed surprised by jdk11.

Btw, that shows the branch merge picked to much. We should support 2.10 and sbt 0.13 in master

Reply:
> We should support 2.10 and sbt 0.13 in master

I think `master` was meant for Play.next which already dropped support for `0.13`.

If we bring `0.13` support back on `master` we'd probably have to do the same on play's `master`.